app/app.html

```diff
@@ -14,7 +17,6 @@
         }
       }());
     </script>
-    <script type="text/javascript" src="http://media.twiliocdn.com/sdk/js/client/v1.3/twilio.min.js"></script>
```

Comment:
Calling is working so I think it's fine for this to be removed. However, twilio is still being listed in our webpack so I don't know if that's why it's still working.